.github/workflows/ci.yml

```diff
@@ -55,3 +32,111 @@ jobs:
+
+    - name: Print Disk Usage Reports
+      run: df ; echo "" ; docker system df
+
+    # - name: ghcr.io/hyperledger/cactus-connector-corda-server
+    #   run: DOCKER_BUILDKIT=1 docker build ./packages/cactus-plugin-ledger-connector-corda/src/main-server/ -f ./packages/cactus-plugin-ledger-connector-corda/src/main-server/Dockerfile
+
+    - name: Print Disk Usage Reports
+      run: df ; echo "" ; docker system df
+
+    # - name: ghcr.io/hyperledger/cactus-connector-fabric
+    #   run: DOCKER_BUILDKIT=1 docker build ./packages/cactus-plugin-ledger-connector-fabric/ -f ./packages/cactus-plugin-ledger-connector-fabric/Dockerfile
+
+    - name: Print Disk Usage Reports
+      run: df ; echo "" ; docker system df
+
+    - name: ghcr.io/hyperledger/cactus-connector-besu
+      run: DOCKER_BUILDKIT=1 docker build ../packages/cactus-plugin-ledger-connector-besu/ -f ./packages/cactus-plugin-ledger-connector-besu/Dockerfile
+
+    - name: Print Disk Usage Reports
+      run: df ; echo "" ; docker system df
+
+    - name: ghcr.io/hyperledger/cactus-besu-all-in-one
+      run: DOCKER_BUILDKIT=1 docker build ./tools/docker/besu-all-in-one/ -f ./tools/docker/besu-all-in-one/Dockerfile
+
+    - name: Print Disk Usage Reports
+      run: df ; echo "" ; docker system df
+
+    - name: ghcr.io/hyperledger/cactus-corda-all-in-one
+      run: DOCKER_BUILDKIT=1 docker build ./tools/docker/corda-all-in-one/ -f ./tools/docker/corda-all-in-one/Dockerfile
+
+    - name: Print Disk Usage Reports
+      run: df ; echo "" ; docker system df
+
+    - name: ghcr.io/hyperledger/cactus-fabric2-all-in-one
+      run: DOCKER_BUILDKIT=1 docker build ./tools/docker/fabric-all-in-one/ -f ./tools/docker/fabric-all-in-one/Dockerfile_v2.x
+
+    - name: Print Disk Usage Reports
+      run: df ; echo "" ; docker system df
+
+    - name: ghcr.io/hyperledger/cactus-fabric-all-in-one
+      run: DOCKER_BUILDKIT=1 docker build ./tools/docker/fabric-all-in-one/ -f ./tools/docker/fabric-all-in-one/Dockerfile_v1.4.x
+
+    - name: Print Disk Usage Reports
+      run: df ; echo "" ; docker system df
+
+    - name: ghcr.io/hyperledger/cactus-quorum-all-in-one
+      run: DOCKER_BUILDKIT=1 docker build ./tools/docker/quorum-all-in-one/ -f ./tools/docker/quorum-all-in-one/Dockerfile
+
+    - name: Print Disk Usage Reports
+      run: df ; echo "" ; docker system df
+
+    - name: ghcr.io/hyperledger/cactus-rust-compiler
+      run: DOCKER_BUILDKIT=1 docker build ./tools/docker/rust-compiler/ -f ./tools/docker/rust-compiler/Dockerfile
+
+    - name: Print Disk Usage Reports
+      run: df ; echo "" ; docker system df
+
+    - name: ghcr.io/hyperledger/cactus-rust-compiler
+      run: DOCKER_BUILDKIT=1 docker build ./tools/docker/test-npm-registry/ -f ./tools/docker/test-npm-registry/Dockerfile
+
+    - name: Print Disk Usage Reports
+      run: df ; echo "" ; docker system df
+
+    - name: ghcr.io/hyperledger/cactus-whitepaper
```

Comment:
@petermetz Why does this workflow need the build process of whitepaper? I'm sure that it is enough to supply whitepaper.md on GitHub.

Reply:
@takeutak Currently if a commit were to introduce a bug into the whitepaper's builder image (the Dockerfile) that does not get noticed at review time then it would get merged in a broken state. I'm trying to get us to a point where as much of the code as possible is covered by the CI for at least on a sanity check/smoke test level. Case in point, the whitepaper build process was broken before and it went unnoticed for months.

Reply:
@petermetz Thanks for replying. So pdf whitepaper has the same contents as md whitepaper, is it right? I'd like to view the pdf file so would you tell me about its URL? After confirming the pdf, I will approve this PR.

Reply:
>  So pdf whitepaper has the same contents as md whitepaper, is it right?

@takeutak That's correct, yes.

> I'd like to view the pdf file so would you tell me about its URL?

There is no URL just yet (it's not published anywhere). The only way to view the pdf file currently is to build it manually with the help of the container.

> I'm concern about the contents of the pdf, so would you wait for my review until I confirm it?

This PR does not change anything in the whitepaper. You can go and review the whitepaper.md (or build the pdf) if you'd like to, but everything in there has been reviewed by you before so I'd say it would be a redundant exercise to re-review the whitepaper. My recommendation is that you review the changes that are being proposed in this pull request (has nothing to do with the contents of the whitepaper).

Reply:
@petermetz Thanks for your explanation. Therefore I approved it.

Reply:
> your explanation, I approved it.

@takeutak No worries, thank you very much!